src/Cake.Common.Tests/Unit/Tools/DotNet/Package/Add/DotNetPackageAdderTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Cake.Common.Tests.Fixtures.Tools.DotNet.Package.Add;
+using Cake.Common.Tests.Fixtures.Tools.DotNet.SDKCheck;
+using Cake.Common.Tests.Fixtures.Tools.DotNet.Workload.Restore;
+using Cake.Common.Tools.DotNet;
+using Cake.Common.Tools.DotNet.Package.Add;
+using Cake.Testing;
+using Xunit;
+
+namespace Cake.Common.Tests.Unit.Tools.DotNet.Package.Add
```

Comment:
What do you think, that path is correct? Then we'll able to add Package.List and Package.Remove commands